terraform/groups/environments/locals.tf

```diff
@@ -0,0 +1,9 @@
+locals {
+
+  images_bucket_arn = data.aws_s3_bucket.image_bucket.arn
+  images_bucket_id = data.aws_s3_bucket.image_bucket.id
+
+  signed_bucket_arn = data.aws_s3_bucket.signed_bucket.arn
+  signed_bucket_id = data.aws_s3_bucket.signed_bucket.id
+
+}
```

Comment:
Minor gripe—run this file through `terraform fmt` to keep things neat (in keeping with the rest of the code):

```suggestion
locals {
  images_bucket_arn = data.aws_s3_bucket.image_bucket.arn
  images_bucket_id  = data.aws_s3_bucket.image_bucket.id

  signed_bucket_arn = data.aws_s3_bucket.signed_bucket.arn
  signed_bucket_id  = data.aws_s3_bucket.signed_bucket.id
}
```